### PR TITLE
feat(router-store): Make usage of forRoot required (#1662)

### LINKED ISSUE #1662: Router Store: Make usage of forRoot required
You can currently register `@ngrx/router-store` in two ways.

```
StoreRouterConnectingModule
```

and

```
StoreRouterConnectingModule.forRoot(optionalConfig)
```

This change would only make the `forRoot()` option available and removes the default providers from the `NgModule` metadata below.

https://github.com/ngrx/platform/blob/master/modules/router-store/src/router_store_module.ts#L134

Also replaces usage examples to use `forRoot()`

https://github.com/ngrx/platform/blob/master/modules/router-store/src/router_store_module.ts#L125

### PR DESCRIPTION
**Pull request by EnricoVogt**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Closes #1662

## Does this PR introduce a breaking change?

```
[x] Yes
[ ] No
```

### CODE REVIEW

**Comment by EnricoVogt** [comment]
i have changes that now.

**Comment by EnricoVogt** [comment]
I have changes this.

**Comment by brandonroberts** [comment]
The breaking change message should show before/after usage, not changes.

BEFORE:

```ts
@NgModule({
  imports: [
    StoreRouterConnectingModule
  ]
})
```

AFTER:

```ts
@NgModule({
  imports: [
    StoreRouterConnectingModule.forRoot()
  ]
})
```

File: modules/router-store/src/router_store_module.ts [hunk 1]
```diff
@@ -122,31 +122,15 @@ enum RouterTrigger {
...
-      useFactory: _createRouterConfig,
-      deps: [_ROUTER_CONFIG],
-    },
-    {
-      provide: RouterStateSerializer,
-      useClass: DefaultRouterStateSerializer,
-    },
-  ],
-})
+@NgModule()
```
Comment (brandonroberts) [inline modules/router-store/src/router_store_module.ts]: It should be an empty object even if there's no metadata.

```suggestion
@NgModule({})
```

Comment (EnricoVogt) [reply modules/router-store/src/router_store_module.ts]: Oh. Thanks for your advises. I hope my changes now meet the requirements.

**Pull request merged**